### ISSUE TITLE
3filmy - caly czas to samo?

### ISSUE THREAD

**Issue by gs188** [mbebe/blomqvist#269]
Czy u was tez jesli wejdziecie do kategorii filmy pokazuje sie caly czas ta sama lista filmow? zalogowany badz nie, caly czas to samo.

**Comment by rafalmar1981** [comment]
Tak, dokładnie tak samo jest na zalukaj ale to wszystko na ich stronach. Coś tam wyczytałem, że innym wszystko działa więc nie wiem.

**Comment by mbebe** [comment]
Jak wybierzesz kategorie np. animacja oraz dokumentalny, to klikajac w film nie wyswietlaja sie np. animacje?

**Comment by rafalmar1981** [comment]
no to teraz zgłupiałem czy chodzi o zalukaj czy się machnołeś i chciałeś napisać w hd seans

**Comment by rafalmar1981** [comment]
jeśli chodzi o 3filmy i zalukaj to chodzi wszystko jest cacy tylko coś po stronach jest porąbane

**Comment by gs188** [comment]
Kolega rafal troche mi 'podkradl' watek ;)   Do szefa mbebe:  chodzi o to ze jak wybieramy wyswietl filmy to zawsze jest taka sama lista filmow, jakby nic nowego nie dodawali.  Grac gra, ale jakby nie bylo nic nowego.

**Comment by mbebe** [comment]
W ustawieniach dodatku - mozesz ustawic sposob wyswietlania.

**Comment by rafalmar1981** [comment]
mbebe wtyczka jest cacy ze stroną cos nie tak

**Comment by gs188** [comment]
> W ustawieniach dodatku - mozesz ustawic sposob wyswietlania.

To bylo to, dzieki za pomoc (i cierpliwosc! ;)
